src/index.html

```diff
@@ -1,12 +1,412 @@
 <!doctype html>
-<html lang="en">
+<html class="page" lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__body">
+    <header class="header">
+      <img src="./images/logo.png"
+           alt="moyo_logo"
+           class="header__logo"
+      >
+      <nav class="nav">
+        <ul class="nav__list">
```

Comment:
```suggestion
      <img 
        src="./images/logo.png"
        alt="moyo_logo"
        class="header__logo"
      >
      <nav class="nav">
        <ul class="nav__list">
```

Reply:
![image](https://github.com/mate-academy/layout_catalog/assets/91411692/1e2404a6-ea0f-4f11-ae16-41f49fc74dc5)
![image](https://github.com/mate-academy/layout_catalog/assets/91411692/4b16d6b4-abe1-45cc-a82f-04223de7d604)

Reply:
Check code styles rules - [HERE](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules) & fix in all cases